dmcontent/govuk_frontend.py

```diff
@@ -119,26 +119,31 @@ def dm_list_input(
     return params
 
 
-def govuk_label(question: Question) -> dict:
-
+def govuk_label(question: Question, is_page_heading: bool = True, **kwargs) -> dict:
+    """
+    :param bool is_page_heading: If True, the label will be set to display as a page heading
+    """
     return {
-        # Style the label as a page heading, following the
-        # GOV.UK Design System question pages pattern at
-        # https://design-system.service.gov.uk/patterns/question-pages/
         "classes": "govuk-label--l",
-        "isPageHeading": True,
+        # If is_page_heading is true, style the label as a page heading,
+        # following the GOV.UK Design System question pages pattern at
+        # https://design-system.service.gov.uk/patterns/question-pages/
+        "isPageHeading": is_page_heading,
 
         "for": f"input-{question.id}",
         "text": get_label_text(question),
     }
 
 
-def govuk_fieldset(question: Question) -> dict:
+def govuk_fieldset(question: Question, is_page_heading: bool = True, **kwargs) -> dict:
```

Comment:
I think we should make `is_page_heading` kwargs only?

```suggestion
def govuk_fieldset(question: Question, *, is_page_heading: bool = True, **kwargs) -> dict:
```